stdlib/universe/filter_mixed_empty_test.flux

```diff
@@ -59,7 +59,7 @@ t_filter_mixed_empty = (table=<-) =>
   table
   |> range(start: 2018-05-22T19:53:26Z)
   |> filter(fn: (r) => r._measurement == "diskio")
-  |> filter(fn: (r) => r["_value"] > 1000, onEmpty: "keep")
+  |> filter(fn: (r) => r["_value"] > 1000.0, onEmpty: "keep")
```

Comment:
This should have worked correct? I though we removed the constraints about equal types for now?

Reply:
Oh if we did then I need to rebase. I'll look at this. If it's not done, I can remove that constraint.

Reply:
Updated.